chipsec/cfg/8086/common.xml

```diff
@@ -616,6 +616,10 @@ Common (default) XML platform configuration file
       <field name="LOCK"           bit="30" size="1"  desc="Lock (set automatically on the first SMI)" />
       <field name="DEBUG_OCCURRED" bit="31" size="1"  desc="Debug Occurred (set by hardware)" />
     </register>
+    <register name="MSR_SMM_FEATURE_CONTROL" type="msr" msr="0x4E0" desc="Enhanced SMM Feature Control">

```

Comment:
I don't think this is actually a common msr, may want to update it in the configuration files which it applies.

Reply:
Based on the Intel SDM it seems that the `MSR_SMM_FEATURE_CONTROL` register is available starting from:
* 4th Generation Intel® Core™ Processors (Haswell microarchitecture)
* Atom Processors Based on the Goldmont Microarchitecture

I moved the register definition to the appropriate files. Can you please double-check that I didn't make any mistakes?